docs/building_blocks_for_flexible_content.md

```diff
@@ -418,12 +418,13 @@ This example uses the `blocks_container` for the element type in the second [two
 
 #### Columns layout
 
-Columns layout spreads its blocks out horizontally. Depending on the number of child blocks, each column will take all (1 block), 1/2 (2 blocks), 1/3 (3 blocks), or 1/4 (4 or more blocks) of the width. If there are more than 4 blocks, blocks will wrap around after every fourth block.
+Columns layout spreads its blocks out horizontally. Depending on the value of `column_count`, each column will take all (1 block), 1/2 (2 blocks), 1/3 (3 blocks), or 1/4 (4 or more blocks) of the width. If there are more than `column_count` blocks, blocks will wrap to a new row. `column_count` can be a value between 2-4 inclusive, any other value will render full-width columns. If `column_count` is missing entirely, it will default to 3.
```

Comment:
This is a really nice solution and I think it works well. My only comment would be that I can't imagine a situation where we'd want four columns, unless I'm overlooking something?

Reply:
It's really just there for future proofing 'just in case'. Happy to remove it but I don't see the harm in keeping it as an option.

Reply:
I think unless we have a need for it we should remove it. It looks fairly easy to extend if we do require it in future, but I don't want to have this as an option for laying out pages if we don't do this kind of layout.

Reply:
OK, this only supports 2 or 3 columns now.